srcpkgs/opencl-headers/template

```diff
@@ -1,9 +1,9 @@
-# Template file for 'opencl2-headers'
-pkgname=opencl2-headers
+# Template file for 'opencl-headers'
+pkgname=opencl-headers
```

Comment:
This should be `OpenCL-Headers` if we're going to rename the package, and there should be a meta-package called `opencl2-headers` that pulls this in for backwards compatibility.